geoapi/utils/lidar.py

```diff
@@ -69,14 +69,13 @@ def get_bounding_box_2d(filePaths: List[str]) -> MultiPolygon:
     for input_file in filePaths:
         proj4 = getProj4(input_file)
 
-        las_file = laspy.file.File(input_file, mode="r-")
+        las_file = laspy.read(input_file)
```

Comment:
need to add `lazrs` python package.  looks like in the new laspy a backed needs to be installed.  i installed lazrs and all works well with that locally.  I'm investigating why the unit tests didn't pick that up (maybe the files we use to test this function don't trigger the error: "LaspyException: No LazBackend selected, cannot decompress data" ).

Reply:
okay, i see the issue and added a unit test. we were never testing any compressed files (ie. laz). we had some files that had .laz ending but they were uncompressed.  i got a PR targeting this branch and adding a unit test.  see https://github.com/TACC-Cloud/geoapi/pull/224